[DOCS] Viewer role needs experimental settings for Fleet UI access (#129614)
elastic/elasticsearch: docs/reference/security/authorization/built-in-roles.asciidoc

===============================
* This role provides read access to any index that is not prefixed with a dot.
* This role automatically grants read-only access to new {kib} features as soon as they are available.

> * In version 8.17, access to Fleet UI with this role requires Fleet experimental features to be enabled which are disabled by default. See `xpack.fleet.enableExperimental` in https://www.elastic.co/guide/en/kibana/8.17/fleet-settings-kb.html[Kibana settings documentation,title=Kibana 8.17 settings].

Reviewer: this is a copyedit, but I think we need more info here. the `xpack.fleet.enableExperimental` [seems to take a specific feature flag name](https://github.com/search?q=repo%3Aelastic%2Fkibana%20xpack.fleet.enableExperimental&type=code). the setting is described of "List of experimental feature flag to enable in Fleet" Perhaps I am misreading and it's just a true/false setting

```suggestion
* In version 8.17, access to the Fleet UI with this role requires Fleet experimental features to be enabled. Experimental features are disabled by default. See `xpack.fleet.enableExperimental` in the https://www.elastic.co/guide/en/kibana/8.17/fleet-settings-kb.html[Kibana 8.17 settings reference].
```

Author: In this example, to enable the read-only Fleet access you'd need `xpack.fleet.enableExperimental: ['subfeaturePrivileges']` in your Kibana yaml.
